Backport upstream fix for a crash in back-relay when doing do_search.
RHBZ: #1316450

commit 2e60bf5ed00c1a8794131f53a6c72a78c0766e21
Author: Howard Chu <hyc@openldap.org>
Date:   Wed May 18 12:30:31 2016 +0100

    ITS#8428 init sc_writewait

diff --git a/servers/slapd/back-relay/op.c b/servers/slapd/back-relay/op.c
index 4fdc08a..f7c1fee 100644
--- a/servers/slapd/back-relay/op.c
+++ b/servers/slapd/back-relay/op.c
@@ -97,6 +97,7 @@ relay_back_response_cb( Operation *op, SlapReply *rs )
 		(rcb)->rcb_sc.sc_next = (op)->o_callback;	\
 		(rcb)->rcb_sc.sc_response = relay_back_response_cb; \
 		(rcb)->rcb_sc.sc_cleanup = 0;			\
+		(rcb)->rcb_sc.sc_writewait = 0;			\
 		(rcb)->rcb_sc.sc_private = (op)->o_bd;		\
 		(op)->o_callback = (slap_callback *) (rcb);	\
 }
